src/styles/molecules/_molecules.navigation.scss

```diff
@@ -19,16 +19,23 @@
     min-height: $spacer-lg;
     text-align: center;
 
-    &.is-active {
-      > a,
-      > button {
-        &,
-        &:hover,
-        &:active {
+    > a,
+    > button {
+      &,
+      &:hover,
+      &:active {
+        &.is-active {
```

Comment:
De logica van deze nesting dient omgekeerd te worden (`:hover` in `{class}` en niet omgekeerd)